styles/styles.css

```diff
@@ -707,9 +707,37 @@ main .columns-container {
   opacity: 0;
 }
 
-/* mobile code start here */
+main .why-us h2 {
+  padding-bottom: 20px;
+}
+
+main .why-us p{
+ font-size: 18px!important;
```

Comment:
Using `!important` should generally be avoided as it hinders further cascading of styles. When it seemingly becomes necessary, it's usually a sign of non-ascending specificity in the selectors of rules matching for a given element. I think in this case could be avoided by changing the rule's selector to:
```css
main .section.why-us .default-content-wrapper p { ... }
```
in order to make it more specific than the previously winning rule:
```css
main .section .default-content-wrapper p { ... }
```